indexer/src/main/scala/com/expedia/www/haystack/trace/indexer/config/ProjectConfiguration.scala

```diff
@@ -160,7 +160,7 @@ class ProjectConfiguration extends AutoCloseable {
       username = username,
       password = password,
       consistencyLevel = es.getString("consistency.level"),
-      indexName = es.getString("index.name"),
+      indexPrefixName = es.getString("index.prefix.name"),
```

Comment:
this may not be backward compatible. We can keep the config as 'index.name'. Implementation can either create one index(like we do today) or create daily indices(what we want now).

Reply:
done